InventoryComponents/InventoryGoldCompoent.gd

```diff
@@ -0,0 +1,31 @@
+class_name InventoryGoldComponent
```

Comment:
I just noticed the name of the file is incorrect, it reads "InventoryGoldCompoent"